src/Assets/Asset.php

```diff
@@ -1258,4 +1268,101 @@ public function set_type( string $type ) {
 	public function should_print(): bool {
 		return $this->should_print;
 	}
+
+	/**
+	 * Gets the data for a given compiled Asset.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $src The partial path to the asset.
+	 *
+	 * @return ?array
+	 */
+	public function get_compiled_data( string $src ): ?array {
+		$file = $this->get_root_path() . $this->get_path() . $src . '.asset.php';
```

Comment:
Avoid passing a `$src` here, use the `$this->slug`. Which would change how the rest of this method works a bit. But it should be only one compiled variable per `Asset` instance.

Reply:
I removed this method.